keras_cv/models/segmentation/deeplab_v3_plus/deeplab_v3_plus.py

```diff
@@ -102,13 +107,20 @@ def __init__(
 
         inputs = backbone.input
 
-        backbone_outputs = backbone(inputs)
+        high_level_feature_extractor = keras.Model(
```

Comment:
Can't we use `get_feature_extractor` as in `RetinaNet` ([link](https://github.com/keras-team/keras-cv/blob/b1888aac446e6e49c0bb2bd9ac33586f27eabee5/keras_cv/models/object_detection/retinanet/retinanet.py#L174))?

Reply:
We can, and we should! Done